cirq/testing/random_circuit_test.py

```diff
@@ -12,81 +12,93 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from typing import Optional, Dict, Sequence, Union, cast
-from random import randint, random, sample, choice
+import random
 
 import numpy as np
 import pytest
 
 import cirq
-from cirq.testing.random_circuit import random_circuit, DEFAULT_GATE_DOMAIN
+import cirq.testing
 
 
 def test_random_circuit_errors():
-    with pytest.raises(ValueError):
-        random_circuit(randint(1, 10), randint(1, 10), -1)
-    with pytest.raises(ValueError):
-        random_circuit(randint(1, 10), randint(1, 10), 1.)
+    with pytest.raises(ValueError, match='-1'):
```

Comment:
Should we add more to the matching statement, since this could be any error that prints out this argument?

Reply:
Done